documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -2963,7 +2963,14 @@ endif::community[]
 |[[postgresql-property-toasted-value-placeholder]]<<postgresql-property-toasted-value-placeholder, `+toasted.value.placeholder+`>>
 |`__debezium_unavailable_value`
 |Specifies the constant that the connector provides to indicate that the original value is a toasted value that is not provided by the database.
-If the setting of `toasted.value.placeholder` starts with the `hex:` prefix it is expected that the rest of the string represents hexadecimally encoded octets. See {link-prefix}:{link-postgresql-connector}#postgresql-toasted-values[toasted values] for additional details.
+If the setting of `toasted.value.placeholder` starts with the `hex:` prefix it is expected that the rest of the string represents hexadecimally encoded octets. See {link-prefix}:{link-postgresql-connector}#postgresql-toasted-values[toasted values] for additional details. +
```

Comment:
Let's use the xref link style preferably. I know it's a wild mix of both at this point (historical reasons, ...), but as per what I learned from @roldanbob, we can solely use xref, without any issues for downstream docs processes. So I think we should phase out the link-prefix stuff eventually for the sake of simplicity. No need for a grand scale change just now, but let's not further proliferate its usage.